roles/ceph-osd/tasks/start_osds.yml

```diff
@@ -0,0 +1,18 @@
+---
+- include: osd_fragment.yml
+  when: crush_location
+
+- name: get osd id
+  shell: |
+    ls /var/lib/ceph/osd/ | sed 's/.*-//'
+  changed_when: false
+  failed_when: false
+  always_run: true
+  register: osd_id
+
+- name: start and add that the osd service(s) to the init sequence
```

Comment:
This name is a bit misleading because it does not enable the systemd unit anymore, it's just ensuring that it is started. If we deploy and ceph-volume does not start the OSD this task will fail and notify the user.